docs/labs/installer-getstarted.rst

```diff
@@ -0,0 +1,130 @@
+.. _labs-installing-unofficial-flocker-tools:
+
+==================================
+Getting Started with the Installer
+==================================
+
+#. Installing the Installer:
+
+   First we install the installer on your workstation.
+   This will work on Linux or OS X machines with Docker installed.
+
+   * If you don't have Docker installed, install it now (`Mac       <https://docs.docker.com/mac/started/>`_, `Linux <https://docs.docker.com/linux/started/>`_).
+     Check that Docker is working, for example by running:
+
+     .. prompt:: bash $
+
+        docker ps
+
+     You should get a (possibly empty) list of running containers on your machine.
+
+   * Then install the installer, which will pull the Docker image:
+
+     .. prompt:: bash $
+
+        curl -sSL https://get.flocker.io/ | sh
+
+     This assumes that your user can use ``sudo``, and may prompt you for your password.
+     This installer is a tiny script which puts some wrapper scripts (around ``docker run`` commands) into your :file:`/usr/local/bin`.
+
+   * Now test one of the installed tools:
+
+     .. prompt:: bash $
+
+        uft-flocker-ca --version
+
+     This should return something like ``1.5.0``, showing you which version of the Flocker Client is installed.
+
+#. Make a local directory for your cluster files:
+
+   The tools will create some configuration files and certificate files for your cluster.
+   It is convenient to keep these in a directory, so let's make a directory on your workstation like this:
+
+   .. prompt:: bash $
+
+      mkdir -p ~/clusters/test
+      cd ~/clusters/test
+
+   Now we'll put some files in this directory.
```

Comment:
:grey_question: There was a note here: https://clusterhq.atlassian.net/browse/FLOC-3334 about addressing the comments here: https://github.com/ClusterHQ/unofficial-flocker-tools/issues/38, one of which was about this line. Looks like you are not addressing those points here, which is fine, but I just wanted to call your attention to that in case you wanted to make a new JIRA issue for those adjustments or otherwise put those on your work queue. Concern is that those notes might fall out of our list of things to do if we just close 3334 without addressing those notes.